pagefaultgames/pokerogue
# [Bug] Fix Berserk and Anger Shell from crashing the game when pokemon is forced to switch out

Fixes #1689 

## What are the changes?
- Fixed Berserk and Anger Shell abilities

## Why am I doing these changes?
- Game crashes/freezes when Berserk/Anger Shell is triggered with force switch move targeting the opposing side.

## What did change?
- added optional chain before checking `damage`

### Screenshots/Videos
- Game no longer crashes/freezes

https://github.com/pagefaultgames/pokerogue/assets/68144167/4a96d907-3df5-4119-963c-a62698bf73a5



## How to test the changes?
- Import the session data [here](https://discord.com/channels/1125469663833370665/1125662770629709935/1246535325967319040)
- Kill the first mon, opponent should use Circle Throw then

## Checklist
